### PR TITLE
Fix version copy for missing cli

### LINKED ISSUE #902: Copying version information does not work without CodeQL CLI
**Describe the bug**
When you click the extension information at the top right corner, version information should be copied, see #845.
However, this does not work if no CodeQL CLI is configured (or when the extension fails finding it for some reason).

Since the copied data also contains information about the extension version and platform, it should work without a CodeQL CLI, especially in case an extension bug prevents CodeQL CLI detection from working; otherwise users could not easily report such a bug.

**Version**
Extension:
```
CodeQL extension version: 1.5.1 
CodeQL CLI version: 2.5.7 
Platform: win32 x64
```

VSCode:
```
Version: 1.57.1 (user setup)
Commit: 507ce72a4466fbb27b715c3722558bb15afa9f48
Date: 2021-06-17T13:28:07.755Z
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19041
```

**To reproduce**
1. Open the CodeQL extension settings (`@ext:github.vscode-codeql`)
2. Change "CodeQL > CLI: Executable Path" to an invalid path
3. Click the text "CodeQL" at the bottom right corner
:x: It fails with the message "No distribution found (codeQL.copyVersion)" and no information is copied

**Expected behavior**
Copying version information should be possible without a CodeQL CLI; the copied data should indicate that CodeQL CLI is missing, e.g.:
```
CodeQL extension version: 1.5.1 
CodeQL CLI version: <missing>
Platform: win32 x64
```

### PR DESCRIPTION
**Pull request by mgsium**
Fixes #902 

## Checklist
- [x]  CHANGELOG.md has been updated to incorporate all user visible changes made by this pull request.
- [x] Issues have been created for any UI or other user-facing changes made by this pull request.

### CODE REVIEW

**Comment by mgsium** [comment]
Fixes #902

**Review by shati-patel — COMMENTED** [review]
Woohoo, first PR 😎 

This looks good! Could I get someone else on @github/codeql-vscode-reviewers to review/approve, since we paired on this 🙂

File: extensions/ql-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
 
+- Fix bug where version information does not copy when the CLI is missing. [#958](https://github.com/github/vscode-codeql/pull/958)
```
Comment (edoardopirovano) [inline extensions/ql-vscode/CHANGELOG.md:5]:
```suggestion
- Fixed a bug where copying the version information fails when a CodeQL CLI cannot be found. [#958](https://github.com/github/vscode-codeql/pull/958)
```

**Review by edoardopirovano — APPROVED** [review]
One minor suggestion rephrasing the change note, otherwise LGTM.

**Comment by edoardopirovano** [comment]
And congrats on your first PR! 🎉

**Review by aeisenberg — APPROVED** [review]
Nice work.

**Pull request merged**